packages/block-library/src/post-comments-form/index.php

```diff
@@ -28,6 +28,11 @@ function render_block_core_post_comments_form( $attributes, $content, $block ) {
 	$form               = ob_get_clean();
 	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classes ) );
 
+	// Enqueue the comment-reply script.
+	global $wp_scripts;
+	$wp_scripts->in_footer[] = 'comment-reply';
+	$wp_scripts->queue[]     = 'comment-reply';
```

Comment:
what if we call wp_enqueue_script instead of using the globals directly?

Reply:
That was my initial approach too, however it wasn't working - that's why I used `wp_scripts`.

After your comment I re-tested this and it seems to be working fine so I pushed a commit with the fix.
Weird that it wasn't working when I initially tested it though... the only thing I can think of is I had a typo in the script-handle when initially testing with `wp_enqueue_script` 🤷